Functions/Engines/Wine/script.js

```diff
@@ -111,7 +111,7 @@ Wine.prototype.prefix = function (prefix) {
     }
 
     // set
-    this._prefix = prefix;
+    this._prefix = prefix.replace(/[^a-z0-9_\-\ ]/gi, '');
```

Comment:
Oh ok, I thought the Java `replace` method was used here, therefore I was a little confused, but it seems to be a specific javascript method instead, interesting.